content/docs/04-clusters/03-edge/04-edgeforge-workflow/03.5-palette-canvos.md

```diff
@@ -0,0 +1,658 @@
+  <br />
+
+  ```bash
+  docker run --privileged hello-world
+  ```
+  If the command above fails, create a `docker` user group and add the current user to it using the instructions available [here](https://docs.docker.com/engine/install/linux-postinstall/). Alternatively, if you do not want to create a user group, ensure you can issue Docker commands with `sudo` privileges on your Linux machine.
+
+  </WarningBox>
+
+* A [Spectro Cloud](https://console.spectrocloud.com) account. If you have not signed up, you can sign up for a [free trial](https://www.spectrocloud.com/free-tier/).
+
+
+* Palette registration token for pairing Edge hosts with Palette. You will need tenant admin access to Palette to generate a new registration token. For detailed instructions, refer to the [Create Registration Token](/clusters/edge/site-deployment/site-installation/create-registration-token) guide.
+
+
+# Instructions
+
+Use the following instructions on your Linux machine to create all the required Edge artifacts quickly. 
+<br />
+
+1. Check out the [CanvOS](https://github.com/spectrocloud/CanvOS.git) GitHub repository containing the starter code. 
+  <br />
+
+  ```bash
+  git clone https://github.com/spectrocloud/CanvOS.git
+  ```
+
+
+2. Change to the **CanvOS/** directory. 
+  <br />
+
+  ```bash
+  cd CanvOS
+  ```
+
+
+3. View the available [git tags](https://github.com/spectrocloud/CanvOS/tags).
+<br />
+
+  ```bash
+  git tags
+  ```
+
+
+4. Check out the repository tag that matches your Palette product version. Your Palette product version is displayed in the top left corner, as shown in the screenshot below. 
+
+  ![Screenshot displaying the Palette product version.](/tutorials/palette-canvos/clusters_edge_palette-canvos_palette-version.png)
+
+  It is essential to check out a matching git tag so the generated Edge installer will be compatible with your Palette product version. This guide uses **v3.4.3** tag as an example. 
+  <br />
+
+  ```shell
+  git checkout v3.4.3
+  ```
+
+
+5. Review the files relevant for this guide. 
+    - **.arg** - Defines arguments to use during the build process. 
+    - **Dockerfile** - Bakes the arguments to use in the image.
+    - **Earthfile** - Contains a series of commands to create target artifacts.
+    - **earthly.sh** - Script to invoke the Earthfile, and generate target artifacts.
+    - **user-data.template** - A sample user-data file
+  <br />
+
+
+6. Open the **.arg** file in an editor to change one of the arguments. 
+<br />
+
+  ```bash
+  vi .arg
+  ```
+  <br />
+
+7. In the **.arg** file, change the value of the `CUSTOM_TAG` variable to any lowercase and alphanumeric string. This variable defines the custom image tag for the provider OS images. This guide uses the following custom tag as an example. 
+<br /> 
+
+  ```bash
+  CUSTOM_TAG=palette-learn
+  ```
+  Save the file, and exit the editor. 
+  <br />
```

Comment:
We need to change this. The feedback we got is that it felt weird jumping from the file editor to shell.  We decided to stick with doing all actions in the shell, so updating this value needs to happen without opening the file.